Source/DayViewState.swift

```diff
@@ -17,6 +18,7 @@ public final class DayViewState {
 
   public func move(to date: Date) {
     let date = date.dateOnly(calendar: calendar)
+//    if (date == selectedDate) { return } // TODO: If necessary? Just trying to make sure that willMoveTo not be called if we are already on this page
```

Comment:
I suggest against including such statements. If `willMoveTo` is called multiple times, it might indicate an error. Errors should be visible as early as possible.

Returning early if date hasn't changed would only complicate debugging. I'm for very simple `DayViewState` with as little logic as possible.